lens/client.go

```diff
@@ -0,0 +1,43 @@
+package lens
+
+import (
+	"context"
+
+	pb "github.com/RTradeLtd/grpc/lens"
+	"google.golang.org/grpc"
+)
+
+// Client is a lens client used to make requests to the Lens gRPC server
+type Client struct {
+	GC *grpc.ClientConn
+	IC pb.IndexerAPIClient
+}
+
+// NewClient is used to generate our lens client
+func NewClient(url string, insecure bool) (*Client, error) {
+	var (
+		gconn *grpc.ClientConn
+		err   error
+	)
+	if insecure {
+		gconn, err = grpc.Dial(url, grpc.WithInsecure())
+	}
+	if err != nil {
+		return nil, err
+	}
+	indexerConn := pb.NewIndexerAPIClient(gconn)
+	return &Client{
+		GC: gconn,
+		IC: indexerConn,
```

Comment:
this will null ptr very badly if `insecure` is false

on the `orchestrator` branch of the grpc repo I've added utilities for attaching credentials to requests https://github.com/RTradeLtd/grpc/blob/orchestrator/utils/credentials.go, there are examples to using this in the ipfs-orchestrator for now

i guess its fine for now though

Reply:
Nope it should be done right away, it removes the chance of forgetting to  do so before we launch. I'll message you on slack when this PR is ready to go.